packages/core/variables/_colors.scss

```diff
@@ -1,29 +1,45 @@
-// Primærfarger
+// Hovedfarger
 $svart: #000;
-$matt-svart: #3d3d3d;
-$varm-fjellgra: #696666;
-$himmelgra: #f0f0f0;
-$snohvit: #fafafa;
 $varde: #ded4c7;
+$hvit: #fff;
 
-// Sekundær- og illustrasjonsfarger
-$sol: #ffc880;
-$sand: #dcb28b;
-$jord: #765d4b;
-$hav: #0a3166;
-$skog: #0d5745;
-$blomst: #ff7059;
+// Gråtoner
+$gra-10: #f8f8f8;
+$gra-20: #ebebeb;
+$gra-30: #d6d6d6;
+$gra-40: #b8b8b8;
+$gra-50: #999999;
+$gra-60: #6f6f6f;
+$gra-70: #525252;
+$gra-80: #3d3d3d;
+$gra-90: #292929;
+$gra-100: #181818;
 
-$helhvit: #ffffff;
-$lys-gra: #e0dfdd;
-$mellomgra: #bab8b6;
+// Vardetoner
+$varde-20: #f9f6f4;
+$varde-40: #f4f0ec;
+$varde-60: #ede7e0;
+$varde-80: #e5ddd3;
+$varde-120: #cfc1ae;
+$varde-140: #baa589;
+$varde-160: #ab9271;
+$varde-180: #8d7453;
+$varde-200: #68563e;
```

Comment:
Går ut fra at disse navnene er for å holde varde på verdi 100, men jeg vet ikke om jeg er superfan.. Særlig når jeg ser 200 tenker jeg umiddelbart at vi er på en 100, 200, 300, etc. skala.

Vet for så vidt ikke om jeg har en mye bedre idé annet enn å ha skalaen fra 0 til 100 og ha varde som en farge et sted inni der.

Reply:
Jeg er forsåvidt enig i at den burde matche `gra` sin skala.

varde-10
varde-20
varde-30
varde-40
varde
varde-60
varde-70
varde-80
varde-90
varde-100

Tanker, @Murstam?

Reply:
Tenker at man fint kan ha både `$varde` og `$varde-50` som variabler og at de er like

Reply:
Ja, det er for å holde Varde på 100, men er ikke 100% fornøyd selv, men beste jeg kom på der og da. Ja, er jo et alternativ at Varde har to variabler, men er ikke helt optimalt det heller. Må tenke litt på det.

Reply:
Jeg ser at både Carbon og Atlassian har fargeskalaer som går fra 0 (hvitt) til 100 (pluss K = sort), der 100 er nesten sort. Så er hovedfargene bare punkter på disse skalaene. Carbons hovedfarge er f.eks. [Blue-60](https://www.carbondesignsystem.com/guidelines/color/overview#color-anatomy), og Atlassians [er B400](https://atlassian.design/guidelines/brand/color).

Reply:
Ok, men da prøver vi det. Virker som det beste alternativet. Da kjører vi Varde som to variabler, $varde og $varde-50.